tasks/terraform_output.rb

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative '../../ruby_task_helper/files/task_helper.rb'
+require_relative '../lib/cli_helper.rb'
+require 'json'
+
+class TerraformOutput < TaskHelper
+  def output(opts)
+    cli_opts = %w[-no-color -json]
+    dir = File.expand_path(opts[:dir], opts[:_boltdir]) if opts[:dir]
+    cli_opts << "-state=#{File.expand_path(opts[:state], dir)}" if opts[:state]
+    cli_opts = cli_opts.join(' ')
+
+    stdout_str, stderr_str, status = if opts[:dir]
+                                       dir = File.expand_path(opts[:dir], opts[:_boltdir])
```

Comment:
Why re-expand here? This seems like it can switch on `dir` instead of `opts[:dir]` and use the expansion from above

Reply:
Copy paste error. Removed.